Documentation/manual/kcc-design/customization-user-story.md

```diff
@@ -0,0 +1,138 @@
+**Expanding on ‘Customization User’ Scenarios**
```

Comment:
Use hierarchal titles for the document so they convert to HTML for easy reading!

like this:

```markdown
# Title

info here

## Section 1

more info...

### Section 1.1

## Section 2

```